send_email_messages/frontend/refresh-queue.js

```diff
@@ -61,11 +66,78 @@ async function formatMessageRecord(messageType, delivery, volunteer, audience) {
 }
 
 // Compute a list of new queued messages to create, and return it.
-async function computeMessagesToCreate(addWarning) {
+async function computeMessagesToCreate() {
+  const deliveryMessages = await computeDeliveryMessages();
+  const volunteerMessages = await computeVolunteerWelcomeMessages();
+
+  return {
+    messagesToCreate: [
+      ...deliveryMessages.messagesToCreate,
+      ...volunteerMessages.messagesToCreate,
+    ],
+    warnings: [...deliveryMessages.warnings, ...volunteerMessages.warnings],
+  };
+}
+
+// compute a list of volunteer welcome messages to send
+async function computeVolunteerWelcomeMessages() {
+  const EMAIL_TYPE_NAME = "Initial Confirmation: Volunteer";
```

Comment:
If you export this constant at the top of this file, I _believe_ you can import it in send-messages.js below (don't quote me on that...)

Reply:
yep good call!